package.json

```diff
@@ -40,8 +40,8 @@
       "source": "./src/index.ts",
       "require": "./dist/index.cjs",
       "node": {
-        "module": "./dist/index.js",
-        "import": "./dist/index.cjs.js"
+        "import": "./dist/index.cjs.js",
```

Comment:
I agree with changes of `module` ordering 👍🏼 But worth to note that this is still not a valid condition as `import` field` is pointing to a CommonJS entry. Using an ESM stub + `require` field could be nice (internally using `createModule` to reexport)

Reply:
confusingly, despite the name, it's not a CJS entry:

https://unpkg.com/browse/@sanity/client@6.4.9/dist/index.cjs.js